deploy/Dockerfile_debug

```diff
@@ -30,15 +30,8 @@ RUN cat /goarch
 
 
 # Get GCR credential helper
-RUN GOARCH=$(cat /goarch) && CGO_ENABLED=0 && \
-  (mkdir -p /go/src/github.com/GoogleCloudPlatform || true)                  && \
-  cd /go/src/github.com/GoogleCloudPlatform                                  && \
-  git clone https://github.com/GoogleCloudPlatform/docker-credential-gcr.git && \
-  cd /go/src/github.com/GoogleCloudPlatform/docker-credential-gcr            && \
-  git checkout 4cdd60d0f2d8a69bc70933f4d7718f9c4e956ff8                      && \
-  go get -u -t ./...                                                         && \
-  go build -ldflags "-linkmode external -extldflags -static" -i -o /usr/local/bin/docker-credential-gcr main.go
-
```

Comment:
@kun-lu20 Can you make these changes to `deploy/Dockerfile` and `deploy/Dockerfile_warmer`

Reply:
@tejal29 I see. However, currently `docker-credential-gcr` only has prebuilt binaries for amd64 and arm64, we have to build it from source for s390x and other archs. Since s390x support will be added to `executor` and `warmer` images, these changes couldn't be made to `deploy/Dockerfile` and `deploy/Dockerfile_warmer` at this moment.

If we want to keep the dockerfiles consistent, for now we could revert these changes in deploy/Dockerfile_debug so that `docker-credential-gcr` will also be built from source when building `executor:debug` image.

Please let me know what you think. Thanks!

Reply:
The credential helpers are now built from source using the image's architecture, so this should not be an issue anymore. If you're still interested in getting this merged please let me know and I'll take a look.

Reply:
If you rebase this change on `master`, you should only need to change `platforms` in `.github/workflows/images.yaml`.

Reply:
Hi @imjasonh 

Thank you very much! Sorry I was on vacation in the last two weeks. I've rebased this change and added s390x support to `platforms` in `.github/workflows/images.yaml`. Please take a look when you have some time.

We are looking forward to adding s390x support to all kaniko images in the new release. Thanks again!

Reply:
No worries. It looks like you're missing it for the slim image?

Reply:
Thanks @imjasonh ! s390x support of the slim image had been added to the yaml file since v1.7.0, so we do not need to do it this time.